### PR TITLE
Show 'Open Service' badge and popover

### PR DESCRIPTION
**Pull request by ochosi**
As part of our 'Open Source Services' initiative, we'll add a badge that indicates that our service is open. The popover will give additional context as well as a link to our architectural docs.

image

### CODE REVIEW

**Comment by lucasgarfield** [comment]
The Travis CI is failing due to 3 minor linting errors.

You can use `npm run lint` locally to double check before issuing a PR. You can also use `npm run lint:js:fix`.

```
/home/travis/build/RedHatInsights/image-builder-frontend/src/Components/LandingPage/LandingPage.js

   1:11  error  'COMMITHASH' is defined but never used                                                                                              no-unused-vars

   6:9   error  Replace `·HelpIcon,·CodeIcon,·ExternalLinkAltIcon·` with `⏎··HelpIcon,⏎··CodeIcon,⏎··ExternalLinkAltIcon,⏎`                         prettier/prettier

  53:61  error  Replace `⏎··················code·is·inspectable.·Explore⏎·················` with `·code·is⏎··················inspectable.·Explore`  prettier/prettier
```

**Comment by lucasgarfield** [comment]
Regarding the spacing of the icons, let's add a new class to `LandingPage.scss`. Maybe something like
```
.header-button {
    padding-right: 0;
}
```
Then we can just pass that into the `className` prop for the button components. Easy fix that will work well even if we add more buttons in the future.

File: src/Components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -34,30 +34,49 @@ class LandingPage extends Component {
...
+                  Repositories
                 </Button>
               </TextContent>
             }
           >
             <Button
               variant="plain"
-              aria-label="About image builder"
+              aria-label="About Open Services"
               className="pf-u-pl-sm"
```
Comment (lucasgarfield) [inline src/Components/LandingPage/LandingPage.js:77]: `className="pf-u-pl-sm header-icon"`

File: src/Components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -34,30 +34,49 @@ class LandingPage extends Component {
...
                 </Text>
                 <DocumentationButton />
-                <br />
+              </TextContent>
+            }
+          >
+            <Button
+              variant="plain"
+              aria-label="About image builder"
+              className="pf-u-pl-sm"
```
Comment (lucasgarfield) [inline src/Components/LandingPage/LandingPage.js:43]: `className="pf-u-pl-sm header-button"`

**Comment by ochosi** [comment]
@lucasgarfield Thank you so much for the review! I have folded your suggestions around padding into the first commit. The dropping of the `COMMIT_HASH` variable led to more changes, so I kept that as a separate commit to make it easier to review.

I hope it's better now!

**Comment by lucasgarfield** [comment]
Oh wow, I didn't expect removing COMMITHASH to have so many cascading effects but looking at your commit it makes sense.

Great work!!!

**Review by lucasgarfield — APPROVED** [review]

**Pull request merged**